commit 85d44b601b48d7caf396f6b7855a9d1eb7e5f93e
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-05-10T13:21:56-06:00 (5 months ago)
    
    MANTA-4262 Final sync up of RPC function names in node-boray

diff --git a/lib/buckets.js b/lib/buckets.js
index fc294fe..0bc9642 100644
--- a/lib/buckets.js
+++ b/lib/buckets.js
@@ -51,7 +51,7 @@ function createBucket(rpcctx, owner, bucket, vnode, callback) {
      */
     rpc.rpcCommonBufferData({
         rpcctx: rpcctx,
-        rpcmethod: 'putbucket',
+        rpcmethod: 'createbucket',
         rpcargs: [arg],
         ignoreNullValues: true,
         log: log
diff --git a/lib/objects.js b/lib/objects.js
index 7d72eff..64b443e 100644
--- a/lib/objects.js
+++ b/lib/objects.js
@@ -109,7 +109,7 @@ function createObjectNoVnode(rpcctx, owner, bucket_id, name, content_length,
     rpc.rpcCommonBufferData({
         'rpcctx': rpcctx,
         'log': log,
-        'rpcmethod': 'createObject',
+        'rpcmethod': 'createobject',
         'rpcargs': args
     }, function (err, data) {
         if (!err && data.length > 1) {
